src/HomeKit/HMCompat.cs

```diff
@@ -113,21 +112,10 @@ public partial class HMAccessorySetupManager {
 
 #if !XAMCORE_5_0
 #if __IOS__ && !__MACCATALYST__
-#if NET
-	[EditorBrowsable (EditorBrowsableState.Never)]
-	[Obsolete ("This class is removed.")]
-#endif
-	public unsafe partial class HMAccessorySetupManager : NSObject {
-		public override NativeHandle ClassHandle => throw new InvalidOperationException (Constants.RemovedFromHomeKit);
-
-		public HMAccessorySetupManager () : base (NSObjectFlag.Empty) => throw new InvalidOperationException (Constants.RemovedFromHomeKit);
-		protected HMAccessorySetupManager (NSObjectFlag t) : base (t) => throw new InvalidOperationException (Constants.RemovedFromHomeKit);
-		protected internal HMAccessorySetupManager (NativeHandle handle) : base (handle) => throw new InvalidOperationException (Constants.RemovedFromHomeKit);
+	public unsafe partial class HMAccessorySetupManager {
 
 		public virtual void AddAndSetUpAccessories (HMMatterTopology topology, Action<NSError> completion) => throw new InvalidOperationException (Constants.RemovedFromHomeKit);
```

Comment:
Maybe we should [Obsolete] each of these methods now?

Reply:
yup agreed, missed that